images/base/Dockerfile.ubuntu

```diff
@@ -1,5 +1,11 @@
 FROM ubuntu:20.04
 
+# Install software-properties-common
+RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -y software-properties-common 
```

Comment:
can this be added to the package list below, in baseline packages? That would reduce the amount of times we need the network (apt-get update) as well as the number of layers in the resulting image. I think this should all done in a single layer, since it's all about package updates/installation.

For example:

```docker
# Install baseline packages
RUN apt-get update && \
    DEBIAN_FRONTEND="noninteractive" apt-get install -y \
      bash \
      build-essential \
      ca-certificates \
      curl \
      docker.io \
      htop \
      locales \
      man \
      python3 \
      python3-pip \
      software-properties-common \
      sudo \
      systemd \
      unzip \
      vim \
      wget && \
    # Install latest Git using their official PPA
    add-apt-repository ppa:git-core/ppa && \
    DEBIAN_FRONTEND="noninteractive" apt-get install -y git
```

Reply:
Yea, that SGTM. It was placed above as otherwise `add-apt-repository` command will not be found. But I like what you have above.

Do you mind if we alphabetize the packages too? I can't stand the unordered list lol

Reply:
yeah I sorted the list in my suggestion above, I guess we are similarly neurotic